posthog/models/kafka_engine_dlq/sql.py

```diff
@@ -0,0 +1,21 @@
+KAFKA_ENGINE_DLQ_BASE_SQL = """
+CREATE MATERIALIZED VIEW {database}.{kafka_table_name}_dlq
```

Comment:
Let's make every sql.py file name the table explicitly rather than "interpolate" the _dlq at the end.

Reasoning: If I'm debugging something in the schema I don't want to be jumping through N files to find out what the actual table name is. A little more repetition in code leads to better debuggability if we don't try to make this DRY.